README.md

```diff
@@ -19,7 +19,8 @@ None.
 |----------|-------------|---------|----------|
 | clamav_clamd_configuration | A dictionary of values to set in the clamd configuration file. | `{}` | No |
 | clamav_configuration_backup | Whether or not to backup configuration files before changing. | `false` | No |
-| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |
+| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be `custom` or an ansible.builtin.cron [special_time](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/cron_module.html#parameter-special_time). | `weekly` | No |
+| clamav_cron_custom    | If frequency is set to custom a dictionnary define the timer. | `05:30` | No |
```

Comment:
This cleans up the description, fixes a typo, and updates the correct default value.
```suggestion
| clamav_cron_custom    | If frequency is set to `custom`, a dictionary to define the timer. | `{"day": "*", "job": "/usr/local/share/virus_scan.sh", "minute": "30", "month": "*", "hour": "5", "weekday": "*"}` | No |
```